config/locales/en.yml

```diff
@@ -1992,6 +1992,9 @@ en:
       cav_warning:
         header: 'Warning: conferences and views were not injected'
         message: 'The provider has claimed conferences and views on this claim, but they have not been injected into CCR. Please enter them manually.'
+      scheme_12_warning:
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3478/files#r476385484)

Reply:
👍 have amended naming to scheme_12_fees_warning